core/camel-base-engine/src/main/java/org/apache/camel/impl/engine/DefaultContextPluginManager.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.impl.engine;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Supplier;
+
+import org.apache.camel.RuntimeCamelException;
+import org.apache.camel.health.HealthCheckRegistry;
+import org.apache.camel.spi.PluginManager;
+
+public class DefaultContextPluginManager implements PluginManager {
+    private final Map<Class<?>, Object> extensions = new ConcurrentHashMap<>();
+    private HealthCheckRegistry hcr;
```

Comment:
is this in use?

Reply:
Oh man, I forgot to remove this. This variable is for the next step. Thanks for pointing it.